modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ddl/DdlToCatalogCommandConverter.java

```diff
@@ -84,9 +82,9 @@ static CatalogCommand convert(DropTableCommand cmd) {
     }
 
     static CatalogCommand convert(CreateZoneCommand cmd) {
-        // TODO: IGNITE-19719 We need to define the default engine differently and the parameters should depend on the engine
-        String engine = Objects.requireNonNullElse(cmd.dataStorage(), DEFAULT_STORAGE_ENGINE);
-        String dataRegion = (String) cmd.dataStorageOptions().getOrDefault("dataRegion", DEFAULT_DATA_REGION);
+        if (cmd.storageProfiles() == null) {
```

Comment:
Don't we check this stuff inside the command itself?

Reply:
Right, check was removed